dvtemplates/build_k8s_cluster.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -ex
+
+# Build a K8s cluster from kubevirtci
+cd "${PWD}"/../kubevirtci
```

Comment:
May just want to put the entire string within the quotes so it's clear it's a single value.
eg: `cd "${PWD}/../kubevirtci"`

Reply:
Done